app/content-scripts/rewards/index.jsx

```diff
@@ -190,9 +187,9 @@ class RewardsApp {
 			this.reward = request.reward;
 			this.conf = request.conf;
 		}
-		if (document.readyState === 'complete' || document.readyState === 'interactive') {
+		setTimeout(() => {
```

Comment:
@trickpattyFH20 what's up with this setTimeout?

Reply:
React sees the element as non interactive and overwrites the click handlers. I added a comment documenting the setTimeout workaround